addons/ModelViewer3D/documentation.md

```diff
@@ -1,7 +1,7 @@
 ## Description
 
-##### Model Viewer 3D allows for displaying 3D models. <br> Please note that when using a mobile device, an additional <img src="/file/serve/6093598547116032" style="border: 0; display:inline; margin: 0 2px; box-shadow: none" alt="AR"> icon will be visible in the vicinity of the 3D model. Interacting with the icon allows switching the preview to the Augmented Reality mode ([https://modelviewer.dev/examples/augmentedreality](https://modelviewer.dev/examples/augmentedreality "https://modelviewer.dev/examples/augmentedreality")).
-
+##### Model Viewer 3D allows for displaying 3D models. <br> Please note that when using a mobile device, an additional <img src="/file/serve/6093598547116032" style="border: 0; display:inline; margin: 0 2px; box-shadow: none" alt="AR"> icon will be visible in the vicinity of the 3D model. Interacting with the icon allows switching the preview to the Augmented Reality mode ([https://modelviewer.dev/examples/augmentedreality](https://modelviewer.dev/examples/augmentedreality "https://modelviewer.dev/examples/augmentedreality"))
+##### Model Viewer 3D works on iPhones from iOS version 14. Supports Augmented Reality (AR) session in mobile device, in iOS it is necessary to check "Disable navigation panels automatic appearance" in Lesson editor for the AR function to work properly.
```

Comment:
Proponuję aby zostawić tutaj `"Model Viewer 3D works on iPhones from iOS version 14."`
Natomiast resztę (moja propozycja na resztę) przenieść do nowego rzdziały/sekcji np. Augmented Reality (AR) support
`To support an augmented reality (AR) session on mobile devices, on iOS it is necessary to check the "Disable navigation panels automatic appearance" option in the lesson editor for the AR function to work properly.` - nie wstrzymujące